model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -355,23 +351,6 @@ object ImportStatuses {
   case object Error extends ImportStatus
 }
 
-object WorkspaceShardStates {
-  sealed trait WorkspaceShardState extends RawlsEnumeration[WorkspaceShardState] {
-    override def toString = getClass.getSimpleName.stripSuffix("$")
-    override def withName(name: String): WorkspaceShardState = WorkspaceShardStates.withName(name)
-  }
-
-  def withName(name: String): WorkspaceShardState = name.toLowerCase match {
-    case "unsharded" => Unsharded
-    case "sharded" => Sharded
-    case _ => throw new RawlsException(s"invalid ShardState [${name}]")
-  }
-
-  case object Unsharded extends WorkspaceShardState
-  case object Sharded extends WorkspaceShardState
-  case object Unknown extends WorkspaceShardState
-}
-
```

Comment:
we no longer need this, since all workspaces are sharded. Removing `WorkspaceShardStates` is what cascaded a ton of syntax changes elsewhere in the codebase.